### PR TITLE
Adds changes to lp rewards, price range details

### PR DESCRIPTION
**Pull request by candida-d**
This PR adds more explanation for the price range set lp prices, and fixes spelling on various pages

### CODE REVIEW

File: docs/tutorials/committing-liquidity.md [hunk 1]
```diff
@@ -29,18 +29,18 @@ Providing liquidity means you'll need to actively manage your commitment. While
...
+In your liquidity commitment submission, you need to propose a fee factor, as a decimalised number that is converted to percentage. Your proposed fee factor is used, with other proposed fee factors, to determine the market's liquidity fee. Once the fee for the market is set, all liquidity orders charge that fee, regardless of whether the provider's submitted fee was higher or lower, and whether you propose a higher (or lower) factor, you are still a liquidity provider. The proposed fees are used to calculate the actual fee each participant will pay on a trade in that market.
 
 :::note Read more 
 [Liquidity rewards](./../concepts/liquidity/rewards-penalties.md)
 :::
 
-### Shapes for commitment
+### Shapes for commitment [WIP - fix link]
```
Comment (witgaw) [inline docs/tutorials/committing-liquidity.md:38]:
```suggestion
### Shapes for commitment
```

File: docs/concepts/liquidity/provision.md [hunk 2]
```diff
@@ -81,7 +84,7 @@ A liquidity commitment transaction must include:
 :::
 
 ### Orders created from commitment
-In essence, liquidity commitment orders are sets of pegged orders grouped by order book size, with a proportion set for each order within the order 'shape'. The overall volume that Vega will imply and automatically place on the order book from this depends on the remaining liquidity obligation, the best bid / ask prices, and the LP price range set on the market. The reference price, offsets and proportion at the reference / offset can always be amended. 
+In essence, liquidity commitment orders are sets of pegged orders grouped by order book size, with a proportion set for each order within the order 'shape'. The overall volume that Vega will imply and automatically place on the order book from this depends on the remaining liquidity obligation, the best bid / ask prices, and the LP price range set on the market. The reference price, offsets and proportion at the reference / offset can always be amended. If an order created from the commitment ends up with a price outside the [price range set in the market proposal](#price-range-for-liquidity-orders), it's adjusted so it falls on the boundary.
```
Comment (witgaw) [inline docs/concepts/liquidity/provision.md:87]:
```suggestion
In essence, liquidity commitment orders are sets of pegged orders grouped by order book side, with a proportion set for each order within the order 'shape'. The overall volume that Vega will imply and automatically place on the order book from this depends on the remaining liquidity obligation, the best bid / ask prices, and the LP price range set on the market. The reference price, offsets and proportion at the reference / offset can always be amended. If an order created from the commitment ends up with a price outside the [price range set in the market proposal](#price-range-for-liquidity-orders), it's adjusted so it falls on the boundary. 
```

File: docs/concepts/liquidity/rewards-penalties.md [hunk 3]
```diff
@@ -49,22 +50,27 @@ In the example below, there are 3 liquidity providers all bidding for their chos
...
 
-The market's liquidity fee and the trading volume determine how big the liquidity fee pool is, and a provider's equity-like share of the market determines how that pool is distributed.
+The market's liquidity fee and the trading volume determine how big the liquidity fee pool is. A provider's equity-like share of the market and their liquidity score determine how that pool is distributed.
+
+The liquidity score is the average volume-weighted probability of trading of all their orders within the [liquidity order price range](provision.md#price-range-for-liquidity-orders), averaged over the <NetworkParameter frontMatter={frontMatter} param="market.liquidity.providers.fee.distributionTimeStep" hideName={false} />. It's calculated for all orders placed by the liquidity provider: automatically deployed and limit orders. 
+
+Generally speaking, an order's probability of trading decreases the further away from the mid-price it is placed, so all other things being constant, the provider who places orders closer to the mid-price will receive a higher fraction of the fees than someone who places orders further away.
```
Comment (witgaw) [inline docs/concepts/liquidity/rewards-penalties.md:67]:
```suggestion
Generally speaking, an order's probability of trading decreases the further away from the mid-price it is placed, so all other things being constant, the provider who places orders closer to the mid-price will receive a higher fraction of the fees than someone who places orders further away. Furthermore, the probability of trading is set to 0 outside the narrowest price monitoring bounds, so any orders deployed there will decrease the liquidity score.
```

**Review by witgaw — COMMENTED** [review]
Looks good overall, left a few comments/suggestions:

File: docs/concepts/liquidity/provision.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ import NetworkParameter from '@site/src/components/NetworkParameter';
...
+Providing liquidity on a market can be done using a combination of two tactics: [batched limit orders](./../trading-on-vega/orders#batch-order) and automatic orders created from a [liquidity commitment](#liquidity-commitments). Supplying limit orders makes an LP eligible to receive maker fees when volume they place on the book is hit, while a liquidity commitment makes an LP eligible to receive a portion of the liquidity fee from every trade on the book, regardless of if it their volume that was targeted or not. Using a liquidity provision strategy that combines both tactics means the provider is eligible to receive both fees.
```
[the remark this answers is not in the document]
Comment (candida-d) [reply docs/concepts/liquidity/provision.md:11]: I thought that wasn't the case, but you would know, so I've amended your suggestion to cover that.

File: docs/concepts/liquidity/provision.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ import NetworkParameter from '@site/src/components/NetworkParameter';
...
+Providing liquidity on a market can be done using a combination of two tactics: [batched limit orders](./../trading-on-vega/orders#batch-order) and automatic orders created from a [liquidity commitment](#liquidity-commitments). Supplying limit orders makes an LP eligible to receive maker fees when volume they place on the book is hit. A liquidity commitment makes an LP eligible to receive a portion of the liquidity fee from every trade on the book, regardless of if their volume that was targeted or not, and orders that are hit will trigger the maker fee. Using a liquidity provision strategy that combines both tactics means the provider is eligible to receive both fees.
```
Comment (witgaw) [inline docs/concepts/liquidity/provision.md:11]:
```suggestion
Providing liquidity on a market can be done using a combination of two tactics: [batched limit orders](./../trading-on-vega/orders#batch-order) and automatic orders created from a [liquidity commitment](#liquidity-commitments). Supplying limit orders makes an LP eligible to receive maker fees when volume they place on the book is hit. A liquidity commitment makes an LP eligible to receive a portion of the liquidity fee from every trade in the market and orders that are hit will earn the maker fee.
```

**Review by witgaw — APPROVED** [review]

**Pull request merged**